Testscripts/Windows/Power-Hibernate.ps1

```diff
@@ -210,6 +210,12 @@ install_package "ethtool"
 				throw "Did not verify the VM status stopped after hibernation starts"
 			}
 
+			# For POWER-HIBERNATE-SRIOV-DEALLOCATE case
```

Comment:
Can you check again if you need this step again? The previous hibernate command put this VM into deallocating and stopped state. I am not if this command is functional in the test step.

Reply:
Hi @juhlee-microsoft ,
The "hibernate" command only make the VM status become "stopped“ but not "deallocated". IMO, in "stopped" status the VM resources won't be released, and if it start again the VF cards won't be changed. But in "deallocated" status, the VM resources(like temporary disk and VF cards) will be released and assign a new(usually different) one when the VM is starting. That's why this issue happened.

Reply:
Understood. If you put this VM in a de-allocated state, you will need to get the public IP address after resuming and overwrite in $AllVMData.Public. Otherwise, the rest of the procedure will fail.

Reply:
Thanks @juhlee-microsoft ! I'm still working on this part and will update soon.

Reply:
Hi @juhlee-microsoft ,
I just updated this part. I have verified that the following command uses the new public IP address. Please help to review again. Thanks!